build_tools/github_actions/fetch_test_configurations.py

```diff
@@ -117,10 +117,10 @@ def _get_script_path(script_name: str) -> str:
     "rocsparse": {
         "job_name": "rocsparse",
         "fetch_artifact_args": "--blas --tests",
-        "timeout_minutes": 60,
+        "timeout_minutes": 120,
         "test_script": f"python {_get_script_path('test_rocsparse.py')}",
         "platform": ["linux", "windows"],
-        "total_shards": 6,
+        "total_shards": 8,
```

Comment:
For these 120 minute timeouts with 6-8 shards, do we actually have enough test runner capacity to support that? I might prefer to instead cut tests instead of increase these.

We've had long queues (4h+) today in particular for gfx1151 test machines.

I'm concerned that if we merge this and propagate it to rocm-libraries, our queue times will get significantly worse and we'll have a hard time backing that out.